auth-api/src/auth_api/utils/enums.py

```diff
@@ -160,7 +160,6 @@ class OrgStatus(Enum):
 
     ACTIVE = 'ACTIVE'
     INACTIVE = 'INACTIVE'
-    PENDING_AFFIDAVIT_REVIEW = 'PENDING_AFFIDAVIT_REVIEW'
```

Comment:
Should this be replaced by "PENDING_STAFF_REVIEW"?

Reply:
It should already be there...So just have to remove the PENDING_AFFIDAVIT_REVIEW

Reply:
Right, didn't expend the code after these.